src/config/FirebaseConfig.jsx

```diff
@@ -1,6 +1,7 @@
 export const firebaseConfig = {
   apiKey: 'AIzaSyAXDqsWK4quNVaf9-YV2e28NsxkfA9rzJA',
   authDomain: 'dreamerscholars.firebaseapp.com',
+  projectId: 'dreamerscholars',
```

Comment:
These changes shouldn't be a part of this PR

Reply:
I believe this is needed because when I removed it the whole app crashed since it is used in `App.jsx` line 18   `firebase.initializeApp(FirebaseConfig);` for initializing firebase and the `LoginButton` depends on it. If there's a way around plz lmk for now ill keep this file in this PR